tests/instrumentation/transactions_store_tests.py

```diff
@@ -177,7 +177,7 @@ def tracer():
     def queue(event_type, event, flush=False):
         events[event_type].append(event)
 
-    store = Tracer(lambda: frames, lambda frames: frames, queue, Config(), None)
+    store = Tracer(lambda: frames, lambda frames: frames, queue, elasticapm_client.config, elasticapm_client)
```

Comment:
this change was needed so that the tests that use this fixture also respect the default settings we set up in the `elasticapm_client` fixture